dist/colors.scss

```diff
@@ -488,109 +488,101 @@ $blk-235: #060606;
 $blk-240: #000000;
 
 // UI Colors Light Theme
-$ui-blue-1: $h-bb-060;
-$ui-blue-2: $h-bb-050;
-$ui-blue-3: $h-bb-070;
-$ui-green-1: $h-gg-060;
-$ui-green-2: $h-gg-050;
-$ui-green-3: $h-gg-070;
-$ui-yellow-1: $h-yy-060;
-$ui-yellow-2: $h-yy-050;
-$ui-yellow-3: $h-yy-070;
-$ui-red-1: $h-rr-060;
-$ui-red-2: $h-rr-050;
-$ui-red-3: $h-rr-070;
+$ui-brand: $h-bb-060;
+$ui-brand-hover: $h-bb-070;
+$ui-brand-press: $h-bb-080;
 $ui-background: $blk-005;
 $ui-foreground-1: $blk-000;
 $ui-foreground-2: $blk-010;
 $ui-foreground-3: $blk-020;
 $ui-text-1: $blk-220;
 $ui-text-2: $blk-170;
 $ui-text-3: $blk-140;
+$ui-text-inverse: $blk-000;
+$ui-text-link: $h-bb-070;
 $ui-border-1: $blk-050;
 $ui-border-2: $blk-030;
 $ui-border-3: $blk-020;
-$ui-border-4: $blk-090;
-$ui-border-5: $blk-130;
+$ui-border-input: $blk-100;
+$ui-info: $h-bb-070;
+$ui-success: $h-gg-060;
+$ui-warning: $h-yy-060;
+$ui-danger: $h-rr-060;
+$ui-danger-hover: $h-rr-070;
+$ui-danger-press: $h-rr-080;
 
 // UI Colors Dark Theme
-$ui-blue-1-dark: $d-bb-420;
-$ui-blue-2-dark: $d-bb-430;
-$ui-blue-3-dark: $d-bb-410;
-$ui-green-1-dark: $d-gg-420;
-$ui-green-2-dark: $d-gg-430;
-$ui-green-3-dark: $d-gg-410;
-$ui-yellow-1-dark: $d-yy-420;
-$ui-yellow-2-dark: $d-yy-430;
-$ui-yellow-3-dark: $d-yy-410;
-$ui-red-1-dark: $d-rr-420;
-$ui-red-2-dark: $d-rr-430;
-$ui-red-3-dark: $d-rr-410;
-$ui-background-dark: $blk-210;
+$ui-brand-dark: $v-bb-160;
+$ui-brand-hover-dark: $h-bb-060;
+$ui-brand-press-dark: $h-bb-070;
+$ui-background-dark: $blk-190;
 $ui-foreground-1-dark: $blk-200;
-$ui-foreground-2-dark: $blk-190;
-$ui-foreground-3-dark: $blk-180;
+$ui-foreground-2-dark: $blk-210;
+$ui-foreground-3-dark: $blk-220;
 $ui-text-1-dark: $blk-000;
 $ui-text-2-dark: $blk-060;
 $ui-text-3-dark: $blk-090;
+$ui-text-inverse-dark: $blk-220;
+$ui-text-link-dark: $d-bb-420;
 $ui-border-1-dark: $blk-170;
 $ui-border-2-dark: $blk-180;
 $ui-border-3-dark: $blk-190;
-$ui-border-4-dark: $blk-130;
-$ui-border-5-dark: $blk-090;
+$ui-border-input-dark: $blk-130;
+$ui-info-dark: $d-bb-420;
+$ui-success-dark: $d-gg-420;
+$ui-warning-dark: $d-yy-420;
+$ui-danger-dark: $d-rr-420;
+$ui-danger-hover-dark: $v-ro-140;
+$ui-danger-press-dark: $v-ro-160;
```

Comment:
Per discussion in Teams: remove intermediate vars and assign palette variables directly to CSS vars.

Reply:
Looks good to me.

A nice-to-have would be `// #F8F8F8` after each for easy way to see what each color is without having to go look each one up.

Reply:
@TheBlueDog  that is a great idea.

Reply:
`$ui-info-dark: $d-bb-420 // #00A0FF;`

like that? idk how that should look

Reply:
Per @julio8a's comment, looks like we're keeping these intermediate SASS vars.

Reply:
https://github.com/Esri/calcite-colors/pull/63/commits/d0b9b939febd95a93c0b2c00b82cb975fc35083e

updated